packages/worker/src/activity.ts

```diff
@@ -53,6 +53,7 @@ export class Activity {
         }
         return { completed: { result: await this.dataConverter.toPayload(result) } };
       } catch (err) {
+        // TODO: should we only allow CancelledError to count as valid cancellation?
```

Comment:
I think so yes. Not found would mean you timed out or something else bad happened.